package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "concert-vast",
-  "version": "1.3.0",
+  "version": "1.3.1",
   "description": "Simple Vast Parsing for Concert Video Ads",
   "main": "src/index.js",
```

Comment:
@jeninedrew I think this line needs to be updated to `dist/index.js`. Right now, consumers (Hymnal Ad SDK) will are looking at this property, which is the uncompiled version of the library.